data_temp/projects.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
It doesn't look like Prettier formatted this JSON file. Did you run `yarn format`? Or did it not automatically run when you ran `git commit`?

Reply:
It doesn't seem like it ran. It might be because I used the git extension interface in VS Code to commit it. I made a commit that prettifies it.